### PR TITLE
Add vertical plugs

### PR DESCRIPTION
**Pull request by WillNilges**
You can now upload vertical plugs. They must be 200x728. It's also possible to implement other plug sizes with this.

TODO:
- [ ] Fix instructions on upload page :sweat: 
- [ ] Scale vertical plugs to not be fucking enormous on the admin page

### CODE REVIEW

**Comment by WillNilges** [comment]
haha fuck oops Prod plug doesn't know how to filter out the vertical plugs that's so quirky and cute

File: db.go [hunk 1]
```diff
@@ -28,14 +29,16 @@ severity        INTEGER NOT NULL,
 message         TEXT NOT NULL
 );`
 
-const SQL_CREATE_PLUG = `INSERT into plugs (s3id, owner, views, approved)
-VALUES ($1::text, $2::text, $3::integer, false)`
+const SQL_CREATE_PLUG = `INSERT into plugs (s3id, owner, views, approved, shape)
+VALUES ($1::text, $2::text, $3::integer, false, $4::text)`
 
 const SQL_RETRIEVE_APPROVED_PLUGS = `SELECT id, s3id, owner, views FROM plugs WHERE approved=true`
```
Comment (galenguyer) [inline db.go:35]: You need another query to retrieve only "banner" plugs - this is the method currently used by default and it won't filter out non-"banner" plugs. If this query isn't used elsewhere changing it should be fine, but make sure you won't change the behavior of something like the admin page

File: plug.go [hunk 2]
```diff
@@ -12,6 +12,7 @@ type Plug struct {
 	ViewsRemaining int
 	Approved       bool
 	PresignedURL   string
+    Shape          string
```
Comment (galenguyer) [inline plug.go:15]: Is github just rendering this indentation wrong or is it less indented than everything else? gofmt is your friend.

**Review by galenguyer — CHANGES_REQUESTED** [review]
please run gofmt it's so nice :pwetty_pwease:

File: db.go (hunk 1 above)
Comment (WillNilges) [reply db.go:35]: https://github.com/ComputerScienceHouse/csh-plug/pull/8/files#diff-ac94c91f8ea84762d55b6932ef1466a2165984591d3e646360c5bb31b2130900R93-R101

**Review by galenguyer — APPROVED** [review]
LGTM! :shipit:

**Pull request merged**